data/phones/list_phones.py

```diff
@@ -10,7 +10,9 @@
 
 import argparse
 import collections
+import ipapy
```

Comment:
Like at Google, this is a third-party package and should be sorted as such...

Reply:
Done.